lib/src/components/NxTreeViewSelect/NxTreeViewMultiSelect/stateful/NxStatefulTreeViewMultiSelect.tsx

```diff
@@ -7,20 +7,36 @@
 import React, {FunctionComponent, useState} from 'react';
 
 import NxTreeViewMultiSelect from '../NxTreeViewMultiSelect';
-import { Props, propTypes } from './types';
+import { Props, Option, propTypes } from './types';
 export { Props, Option } from './types';
-import useFuzzyFilter from '../../../../util/useFuzzyFilter';
+import useFuzzyFilter, { FuseOptions } from '../../../../util/useFuzzyFilter';
 
 const NxStatefulTreeViewMultiSelect: FunctionComponent<Props> = function NxStatefulTreeViewMultiSelect(props) {
-  const {options} = props,
+  const { options, fuzzyFilterConfig } = props,
       isOpenInitialState = !!props.isOpen;
 
   const [isOpen, toggleOpen] = useState(isOpenInitialState),
       onToggleCollapse = () => {
         toggleOpen(!isOpen);
       };
 
-  const [filteredOptions, filter, setFilter] = useFuzzyFilter(options, {keys: ['name'], threshold: 0.1});
+  /**
+   * Config fuzzyFilter threshold and distance from props.
+   *
+   * Defaults provided to handle our usual max-length for orgs/apps (200):
+   * According to https://fusejs.io/concepts/scoring-theory.html#scoring-theory
+   * an entry is considered a match if it is X characters from location,
+   * where:
+   *  X = distance * threshold,
+   *  location = 0 (default)
+   */
+  const filterConfigOptions: FuseOptions<Option> = {
+    keys: ['name'],
+    threshold: 0.1,
+    distance: 2000,
```

Comment:
These defaults that we provide should probably be documented, so that callers who only partially override them can know what to expect

Reply:
> These defaults that we provide should probably be documented, so that callers who only partially override them can know what to expect

Or alternatively, maybe these defaults should only be used if the `fuzzyFilterConfig` isn't specified at all.